notifications.bs

```diff
@@ -760,42 +782,28 @@ method steps are:
 <ol>
  <li><p>Let <var>global</var> be the <a>current global object</a>.
 
- <li><p>Let <var>permissionDescriptor</var> be the {{PermissionDescriptor}} with
- {{PermissionDescriptor/name}} set to "<code>notifications</code>".
-
- <li><p>Let <var>permissionStatus</var> be the result of
- <a lt="create a PermissionStatus">creating a `PermissionStatus`</a> for
- <var>permissionDescriptor</var>.
-
  <li><p>Let <var>promise</var> be <a for=/>a new promise</a> in <a>this</a>'s <a>relevant Realm</a>.
 
  <li>
   <p>Run these steps <a>in parallel</a>:
 
   <ol>
-   <li><p>Run the <a for="powerful feature">permission query algorithm</a> with
-   <var>permissionDescriptor</var> and <var>permissionStatus</var>.
-
-   <li><p>Let <var>permissionState</var> be <var>permissionStatus</var>'s
-   {{PermissionStatus/state}}.
+   <li><p>Let <var>permissionState</var> be the result of
+   <a>querying the notifications permission state</a>.
 
-   <li><p>If <var>permissionState</var> is {{PermissionState/"prompt"}}, then set
-   <var>permissionState</var> to "<code>default</code>".
+   <li><p><a>Queue a global task</a> on the <a>DOM manipulation task source</a> given
+   <var>global</var> to run these steps:
 
-   <li>
-    <p><a>Queue a global task</a> on the <a>DOM manipulation task source</a> given <var>global</var>
-    to run these steps:
+   <ol>
+    <li><p>If <var>deprecatedCallback</var> is given, then <a for=/>invoke</a>
+    <var>deprecatedCallback</var> with « <var>permissionState</var> ». If this throws an
+    exception, then <a>report the exception</a>.
```

Comment:
```suggestion
    <ol>
     <li><p>If <var>deprecatedCallback</var> is given, then <a for=/>invoke</a> 
     <var>deprecatedCallback</var> with « <var>permissionState</var> ». If this throws an exception, 
     then <a>report the exception</a>.
```

Reply:
(please let me know when annevk-lint is available for vscode!)